spec/engine/matchers/semver_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+
+describe SplitIoClient::Semver do
+  let(:valid_versions) { ["1.1.2", "1.1.1", "1.0.0", "1.0.0-rc.1", "1.0.0-beta.11", "1.0.0-beta.2",
```

Comment:
Why are you not using the csv example? 
The main idea of the csv file is to have the same tests across the SDKs, and if we want to update the file or add more examples, the changes are easier.

Reply:
will do